modules/jupyter-widget/package.json

```diff
@@ -29,13 +29,12 @@
     "prepublishOnly": "webpack -p"
   },
   "dependencies": {
-    "@deck.gl/aggregation-layers": "8.8.0-alpha.6",
-    "@deck.gl/carto": "8.8.0-alpha.6",
-    "@deck.gl/geo-layers": "8.8.0-alpha.6",
-    "@deck.gl/google-maps": "8.8.0-alpha.6",
-    "@deck.gl/json": "8.8.0-alpha.6",
-    "@deck.gl/layers": "8.8.0-alpha.6",
-    "@deck.gl/mesh-layers": "8.8.0-alpha.6",
+    "@deck.gl/aggregation-layers": "^8.8.0",
```

Comment:
Please never manually change dependency versions. These are managed by lerna. When you publish a production release you need to do so from the *-release branch.